CODEOWNERS

```diff
@@ -64,7 +64,12 @@
 /bundles/org.openhab.binding.digiplex/ @rmichalak
 /bundles/org.openhab.binding.digitalstrom/ @MichaelOchel @msiegele
 /bundles/org.openhab.binding.dlinksmarthome/ @MikeJMajor
+<<<<<<< HEAD
+/bundles/org.openhab.binding.dmx/ @J-N-K
+/bundles/org.openhab.binding.dominoswiss/ @Friesoch
+=======
 /bundles/org.openhab.binding.dmx/ @openhab/add-ons-maintainers
+>>>>>>> fa699b807e91b5dc2e8094e590b529a8bb9ffcd7
```

Comment:
Can you resolve this conflict?

Reply:
thanks for that hint, that should not have happend -